src/Explorer/QueryCopilot/QueryCopilotTab.tsx

```diff
@@ -201,7 +201,9 @@ export const QueryCopilotTab: React.FC<QueryCopilotTabProps> = ({ explorer }: Qu
       setShowDeletePopup(false);
       useQueryCopilot.getState().refreshCorrelationId();
       const serverInfo = useNotebook.getState().notebookServerInfo;
-      const queryUri = createUri(serverInfo.notebookServerEndpoint, "generateSQLQuery");
+      const queryUri = userContext.features.enableCopilotPhoenixGateaway
```

Comment:
we need to do the same thing for feedback

Reply:
Resolved